eq-author-api/middleware/identification/index.test.js

```diff
@@ -82,10 +97,19 @@ describe("auth middleware", () => {
         expect(res.send).toHaveBeenCalledWith(401);
       });
 
-      it("should send a 401 response if token is invalid", () => {
-        req.header.mockImplementation(() => "Bearer abc.def.ghi");
+      it("should send a 401 response if token is invalid", async () => {
+        req.header.mockImplementation(() => `Bearer invalid.access.token`);
 
-        middleware(req, res);
+        await middleware(req, res);
+
+        expect(logger.error).toHaveBeenCalled();
+        expect(res.send).toHaveBeenCalledWith(401);
+      });
+
+      it("should send a 401 response if token is valid jwt but invalid token", async () => {
```

Comment:
Could this end with "but invalid signature"?

Reply:
Changed to be more explanatory.